edweiss-app/components/courses/CourseParameters.tsx

```diff
@@ -16,31 +17,53 @@ import FancyTextInput from '../input/FancyTextInput';
 
 
 // Icons
-const nameIcon = 'text-outline';
-const descriptionIcon = 'create-outline';
+export const icons: { [key: string]: IconType } = {
+    nameIcon: 'text-outline',
+    descriptionIcon: 'create-outline',
+};
 
 // Tests Tags
-export const testIDs = {
-
+export const testIDs: { [key: string]: string } = {
+    globalView: 'global-view',
+    goBackOpacity: 'go-back-opacity',
+    closeIcon: 'close-icon',
+    title: 'title',
+    message: 'message',
+    scrollView: 'scroll-view',
+    nameDescriptionSectionView: 'name-description-section-view',
+    nameInput: 'name-input',
+    descriptionInput: 'description-input',
+    sectionInput: 'section-input',
+    creditsComponentView: 'credits-component-view',
+    descreaseCreidtsButton: 'descrease-credits-button',
```

Comment:
What do you mean? A file where every testIds of every files are displayed ?!!?
 Imo I think it not a good idea to do that since those testIds are specific to this file and move them elsewhere is just annoying if we want to modify on testId.